src/libs/runtime-utils.js

```diff
@@ -69,6 +71,11 @@ const generateDiskCostFunction = price => ({ size, status }) => {
 export const persistentDiskCost = generateDiskCostFunction(storagePrice)
 export const persistentDiskCostMonthly = generateDiskCostFunction(monthlyStoragePrice)
 
+const ephemeralExternalIpAddressCost = (numStandardVms, numPreemptibleVms) => {
```

Comment:
If there are multiple parameters/arguments for a function the typical convention in this repo is to put them into an object i.e. 
`{numStandardVms, numPreemptibleVms}` 
that way when calling the function you can be transparent about which piece you are sending an arg for and without worrying about order i.e 
`ephemeralExternalIpAddressCost({numStandardVms: 1}`

Reply:
👍🏼  Thanks for the explanation.

Reply:
TOL: I would allow either in this case. The object is a little more verbose, but it is more explicit as Emily pointed out. This is also especially powerful if you want to specify default values for both parameters. If they're not in an object, you'd have to provide the first value if you wanted a non-default 2nd value. If they're in an object then since, as Emily also said, you can provide them in any order when calling, you can omit specifying anything that you want to accept the default value for.

In this case, it looks like `numPreemptibleVms: 0` might be a reasonable default.

Reply:
This came up during another PR review where my thought was…I like the caller site conciseness of functions with default input parameters but our best practice in the interactive analysis team is to generally avoid default values to make it more difficult for the caller of the function to indirectly pass in a value by mistake that they wouldn't intend to, part of it coming from past pain. Especially since the cost estimate calculations are so convoluted as they are, I think I'd rather err on the side of verbosity.